kedro_airflow_k8s/cli.py

```diff
@@ -109,3 +112,35 @@ def upload_pipeline(ctx, output):
 
     with fsspec.open(f"{output}/{dag_filename}", "wt") as f:
         template_stream.dump(f)
+
+
+@airflow_group.command()
+@click.option(
+    "-o",
```

Comment:
Later on we can consider moving some of this options to a config file, but I'll create a separate issue for that.